### PR TITLE
Updated WorkBundle for TransactionManager. Created WorkUnit class

### PR DESCRIPTION
**Pull request by ssj4ndrew**
WorkBundle will have a list of WorkUnit.

### CODE REVIEW

**Comment by ssj4ndrew** [comment]
Updated WorkBundle to be backwards-compatible?

**Comment by ssj4ndrew** [comment]
updated WorkBundle file methods to modify WorkUnitList. Removed fileNameList in WorkBundle

File: src/test/java/emissary/pickup/WorkUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.UUID;
+
+import org.junit.Test;
+
+public class WorkUnitTest {
+    @Test
+    public void testConstructWorkUnit() {
+        WorkUnit WorkUnit = new WorkUnit("file1");
+        assertEquals("WorkUnit filename should match", WorkUnit.getFileName(), "file1");
```
Comment (fbruton) [inline src/test/java/emissary/pickup/WorkUnitTest.java]: Swap expected and actual results. Update for all assertions.

File: src/test/java/emissary/pickup/WorkUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import static org.junit.Assert.assertNull;
+
+import java.util.UUID;
+
+import org.junit.Test;
+
+public class WorkUnitTest {
+    @Test
+    public void testConstructWorkUnit() {
+        WorkUnit WorkUnit = new WorkUnit("file1");
```
Comment (fbruton) [inline src/test/java/emissary/pickup/WorkUnitTest.java]: Lowercase WorkUnit variable name.

**Pull request merged**